quickstarts/oma-qf/config.yml

```diff
@@ -0,0 +1,43 @@
+id: 
```

Comment:
Hey @khickey-newrelic! No need to have this field here, it will be autogenerated when it's merged!
```suggestion
```